torch/onnx/symbolic_opset9.py

```diff
@@ -373,7 +373,10 @@ def rsub(g, self, other, alpha=None):
 
 
 def mul(g, self, other):
-    return g.op("Mul", self, other)
+    if symbolic_helper._is_bool(self) and symbolic_helper._is_bool(other):
+        return g.op("And", self, other)
```

Comment:
Please add comment explaining this is due to ONNX Mul does not support boolean type inputs

Reply:
Ok